uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -227,6 +211,18 @@ define([
       };
 
       /**
+       * Validates and creates the leave request
+       *
+       * @returns {Promise}
+       */
+      function createRequest () {
```

Comment:
this should not be here, it's in the middle of the public methods, while instead it's a private function